packages/shared/src/hooks/notifications/useBookmarkReminder.spec.tsx

```diff
@@ -0,0 +1,179 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import React from 'react';
+import { addDays, addHours } from 'date-fns';
+import {
+  getNextMonday,
+  BookmarkReminderPreference,
+  useBookmarkReminder,
+} from './useBookmarkReminder';
+import { setBookmarkReminder } from '../../graphql/bookmarks';
+
+jest.mock('../../graphql/bookmarks', () => ({
+  ...(jest.requireActual('../../graphql/bookmarks') as Iterable<unknown>),
+  setBookmarkReminder: jest.fn(),
+}));
+
+const client = new QueryClient();
+let mockedNow: Date;
+
+const Wrapper = ({ children }) => (
+  <QueryClientProvider client={client}>{children}</QueryClientProvider>
+);
+
+describe('useBookmarkReminder hook', () => {
+  beforeEach(() => {
+    process.env.TZ = 'UTC'; // unfortunately this doesn't work. the offset is still there
```

Comment:
What does this break?

Reply:
I was hoping to set some specific TZ to just have some validation that we send 0 offset. Setting the env variable didn't work.